eventmesh-storage-plugin/eventmesh-storage-standalone/src/main/java/org/apache/eventmesh/storage/standalone/broker/MessageQueue.java

```diff
@@ -24,24 +24,27 @@
 
 import com.google.common.base.Preconditions;
 
+import lombok.Getter;
+
 /**
- * This is a block queue, can get entity by offset. The queue is a FIFO data structure.
+ *  Represents a message queue that stores instances of MessageEntity.
  */
```

Comment:
I think the original comment seems to be more clear. It can make developer or user easier and faster understand this class. And the modifications to `put(MessageEntity messageEntity)`, `take()`, `getByOffset(long offset)` are in the same situation.